climbing-the-leaderboard.ts

```diff
@@ -0,0 +1,19 @@
+export function climbingLeaderboard(
+  ranked: number[],
+  player: number[]
+): number[] {
+  const uniqueRank = [...new Set(ranked)];
+
+  let scores: number[] = [];
```

Comment:
The instance of `scores` never changes

```suggestion
  const scores: number[] = [];
```